src/libsync/theme.cpp

```diff
@@ -531,14 +546,47 @@ QIcon Theme::folderOfflineIcon(bool sysTray) const
     return themeIcon(QLatin1String("state-offline"), sysTray);
 }
 
-QColor Theme::wizardHeaderTitleColor() const
+QColor Theme::applicationTitleColor() const
```

Comment:
Really this should have been kept out of this PR. That's one moving piece to many for me on the review end, I'm more likely to miss things due to that.

Reply:
Should we revert that commit?

Reply:
Well, depends how much hassle this is for you... I mean in an ideal world I'd love to this that API change in its own PR, but if that's too annoying to remove it from the history of this one I'm not going to slap you around until you do it. ;-)

Reply:
I then try to revert that